### PR TITLE
bugFix because of wrong routing

### PR DESCRIPTION
**Pull request by MerlindCPohl**
bugFix GI-64-level-1-quiz-implementieren

Fixes the wrong routing in Level 1 

> **Reminder:** The Jira ticket **must be linked in the PR title**  
> Example: `GI-21: Implement user login flow`

## Jira Ticket

Link to the Jira ticket: [GI-XXX](https://techstudyfinder.atlassian.net/browse/GI-XXX)

## Author Checklist

- [ x] Code follows project coding standards and conventions
- [x ] Tests have been written or updated
- [ x] All tests pass locally
- [x ] No sensitive data or secrets are included
- [ x] Documentation (README, comments) has been updated if necessary

## Reviewer Notes

Please verify the following during review:

- The implementation matches the described requirements and Jira ticket
- Code changes are logically correct and maintainable
- No unnecessary dependencies or side effects introduced
- Tests cover critical logic and edge cases
- Naming, structure, and readability meet team standards

## Additional Context (optional)

Add any background information or references that help reviewers understand the changes.

---

_Please ensure all criteria are met before merging._

### CODE REVIEW

File: client/src/services/quizApi.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ const API_BASE_URL = import.meta.env.VITE_API_URL;
 export async function postFilterLevel(
   payload: Level1Payload,
 ): Promise<FilterResponse> {
-  const endpoint = `${API_BASE_URL}/filter`;
+  const endpoint = `${API_BASE_URL}/api/quiz/filter`;
```
Comment (DamarisLi) [inline client/src/services/quizApi.ts]: Ich glaube das hier setzt die Route hinter die aus der .env, das sollte nur /quiz/filter sein glaube ich? Je nachdem, was in die .env soll als base url

File: package-lock.json
[the remark this answers is not in the document]
Comment (MerlindCPohl) [reply package-lock.json]: Das war das Pull-Update. Die Änderungen haben sich ergeben durch das Main-Update.
